tests/service/StructureTests.fs

```diff
@@ -651,11 +651,11 @@ type T() =
 """
     => [ (2, 5, 11, 12), (2, 7, 11, 12)
          (3, 4, 4, 12), (3, 7, 4, 12)
-         (3, 4, 4, 12), (4, 8, 4, 12)
+         (3, 4, 4, 12), (3, 10, 4, 12)
```

Comment:
@auduchinok I winged this test a bit 😅.
In the parser I updated the range of `SynMemberDefn.Member(SynBinding (None, SynBindingKind.Normal, false, false, $1, grabXmlDoc(parseState, 3), valSynData, declPat, None, expr, mWholeBindLhs, DebugPointAtBinding.NoneAtInvisible), m)` (see line `1988`) to only included the `lhs` of the SynBinding and not the entire range `m`.
This is the case with all the other SynBindings, so I thought I could fix this and make it consistent.

So, that leads to a change in `getOutliningRanges` (in `ServiceStructure.fs`) where at this point I'm not really sure what I did is correct. Any thoughts here?

Reply:
We ignore some of these ranges in ServiceStructure in Rider, and from a glance it looks like it won't affect us. Not sure about other users, though.